src/ingest_validation_tools/schema_loader.py

```diff
@@ -374,11 +377,16 @@ def _add_constraints(field, optional_fields, offline=None, names=None):
     if field['name'].endswith('_email'):
         field['format'] = 'email'
 
-    # In the src schemas, set to False to avoid limit on sequences.
+    # In the src schemas, set to False to avoid limit on sequences...
     if field['custom_constraints'].get('sequence_limit', True):
         field['custom_constraints']['sequence_limit'] = 3
     else:
         del field['custom_constraints']['sequence_limit']
+    # ... or to allow "N/A":
+    if field['custom_constraints'].get('forbid_na', True):
+        field['custom_constraints']['forbid_na'] = True
+    else:
+        del field['custom_constraints']['forbid_na']
```

Comment:
Why `del` it?

Reply:
I think I just want to minimize the variations that will be seen downstream: Either it's there, and true, or it's not there at all. Could be argued.